db/migrations/000010_remove_server_namespace_idx.sql

```diff
@@ -0,0 +1,18 @@
+-- +goose NO TRANSACTION 
+-- +goose Up
+-- +goose StatementBegin
+DROP INDEX IF EXISTS attributes@idx_server_namespace CASCADE;
+DROP INDEX IF EXISTS versioned_attributes@idx_server_namespace CASCADE;
```

Comment:
Why are we removing this index?

Reply:
Thanks for catching that, I missed reverting this commit.

It was related to change discussed in https://github.com/metal-toolbox/hollow-serverservice/pull/101#discussion_r915225582